plugins/obs-outputs/librtmp/rtmp.c

```diff
@@ -744,6 +744,25 @@ int RTMP_AddStream(RTMP *r, const char *playpath)
     return idx;
 }
 
+// Returns true if the string needs to be freed
+static bool get_hostname(AVal* host, char** result)
```

Comment:
This felt a little bit weird with the return value as the free/nofree.  Maybe 
```cpp
static const char *get_hostname(AVal* host, bool *needs_free)
```

food for thought